pkg/antctl/raw/test/exec.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2024 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/remotecommand"
+)
+
+type ExecResult struct {
+	Stdout bytes.Buffer
+	Stderr bytes.Buffer
+}
+
+type ExecParameters struct {
+	Namespace string
+	Pod       string
+	Container string
+	Command   []string
+}
+
+func (c *Client) execInPod(ctx context.Context, p ExecParameters) (*ExecResult, error) {
```

Comment:
It doesn't seems worth to have this file and the structs `ExecResult` and `ExecParameters`. Quite some code are added because of this.